### ISSUE TITLE
Cruel sun feels extremely weak

### ISSUE THREAD

**Issue by Arimphae** [ArcPh1r3/PaladinMod#8]
Cruel sun feels incredibly weak, for what it's meant to be. A forced, 5 second channel that you can't cancel (unless you have the dash equipped) that deals damage to you AND allies, and I'm not sure what it scales with either, so late game it's useless

And also you can't do ANYTHING else during it's whole duration. Like, the rest of that I can work with, but being frozen in a super long, helpless animation that whole time? Why? Can at least **that** be changed, please? I've died so many times because of that, because I try to use that skill and it just locks me in as a punching bag for enemies.

Other than that, the character is awesome and by far one my favorite I'd say, I think he's great fun to use (even if I'm a little unhappy about the lunar shards nerf), and can't wait to see more stuff added to him

**Comment by ArcPh1r3** [comment]
lole

**Comment by GalacticRuler** [comment]
very mature response. wow.

this is a very valid criticism of this mod, as it's an interesting skill, that is made completely useless by the animation locking you into it. Just make it fire-and-forget, like Huntress' main special, and be done with it.

**Comment by milkev** [comment]
it would be great if it was a 'activate and leave it be', like her other skills. currently its a suicide tool unfortunately.

**Comment by Arimphae** [comment]
> lole

Why lol? It's genuinely the only real issue I've got with the mod, everything else is pretty fun and I look forward to seeing future additions honestly.

**Comment by ArcPh1r3** [comment]
> very mature response. wow.

cry about it


> it would be great if it was a 'activate and leave it be', like her other skills. currently its a suicide tool unfortunately.

op as fuck, don't count on this happening


> Why lol? It's genuinely the only real issue I've got with the mod, everything else is pretty fun and I look forward to seeing future additions honestly.

because it's essentially a joke skill. "what if i took the grandparent sun, and gave it to paladin?"
it has its pros and its cons just as any other skill does, as well as filling a unique niche next to the other specials- so i'm not concerned about making sure it's as viable as all the others

as far as buffs go the only thing i would consider is more damage and making it proc, but i've no time to look into rewriting the grandparent sun myself so unfortunately that's unlikely to change for a while



> I'm not sure what it scales with either, so late game it's useless

scales with enemy hp, not much but it's something

> And also you can't do ANYTHING else during it's whole duration. Like, the rest of that I can work with, but being frozen in a super long, helpless animation that whole time? Why? Can at least **that** be changed, please?

no, full commitment is funnier. lunar shards work though



tl;dr don't count on any buffs because it's funny

**Comment by Arimphae** [comment]
> > Why lol? It's genuinely the only real issue I've got with the mod, everything else is pretty fun and I look forward to seeing future additions honestly.
> 
> because it's essentially a joke skill. "what if i took the grandparent sun, and gave it to paladin?"
> it has its pros and its cons just as any other skill does, as well as filling a unique niche next to the other specials- so i'm not concerned about making sure it's as viable as all the others
> 
> as far as buffs go the only thing i would consider is more damage and making it proc, but i've no time to look into rewriting the grandparent sun myself so unfortunately that's unlikely to change for a while

You know what?...Fair enough lmao; It **is** pretty entertaining to use, even if it usually results in my death (and that of my friends); We did a triple stack of it at one point

> > And also you can't do ANYTHING else during it's whole duration. Like, the rest of that I can work with, but being frozen in a super long, helpless animation that whole time? Why? Can at least **that** be changed, please?
> 
> no, full commitment is funnier. lunar shards work though
> 
> tl;dr don't count on any buffs because it's funny

Yeah I can completely understand that reasoning, because haha funny meme is definitely the kind of nonsense I pull when I play. In that case, I might run it more, I saw that cheeky 2nd emote on Paladin; I know what he's praising.

With that said, gonna close my ticket and I look forward to seeing what other updates add!

**Comment by Arimphae** [comment]
Before I close the ticket (properly), what's "Pride Flare"? I see it mentioned in the notes several times, but I have no clue what it's actually talking about

**Comment by ArcPh1r3** [comment]
> 
> 
> Before I close the ticket (properly), what's "Pride Flare"? I see it mentioned in the notes several times, but I have no clue what it's actually talking about

cruel sun's ancient scepter upgrade

**Comment by Arimphae** [comment]
> > Before I close the ticket (properly), what's "Pride Flare"? I see it mentioned in the notes several times, but I have no clue what it's actually talking about
> 
> cruel sun's ancient scepter upgrade

Ah! Cool, I only installed that mod today; Alrighty, thanks mate!
